modules/express/src/clientRoutes.ts

```diff
@@ -1704,4 +1705,16 @@ export function setupLightningRoutes(app: express.Application, config: Config):
     promiseWrapper(handleUnlockLightningWallet)
   );
   app.get('/api/v2/:coin/wallet/:id/state', prepareBitGo(config), promiseWrapper(handleGetLightningWalletState));
+  app.post(
+    '/api/v2/:coin/wallet/:id/lightning/invoice',
+    parseBody,
+    prepareBitGo(config),
+    promiseWrapper(handleCreateLightningInvoice)
+  );
+  app.post(
+    '/api/v2/:coin/wallet/:id/lightning/pay',
+    parseBody,
+    prepareBitGo(config),
+    promiseWrapper(handlePayLightningInvoice)
+  );
```

Comment:
`setupLightningRoutes` is supposed contain APIs those work with signer LND. Express without signer LND config also should be allowed to work with invoice APIs. They can be added directly within `setupAPIRoutes`.